### ISSUE TITLE
Documentation still shows --secrets-encryption as experimental

### ISSUE THREAD

**Issue by evilhamsterman** [k3s-io/k3s#6786]
Per #6581 the `--secrets-encryption` flag is no longer experimental, and the docs for managing secrets encryption doesn't mention it as experimental. But the reference for the Server Configuration still lists it under Experimental Options https://docs.k3s.io/reference/server-config#experimental-options

**Comment by brandond** [comment]
Yes, the CLI flags have not been sync'd to the docs in a bit. The main secrets-encryption documentation no longer lists the feature as experimental though.

https://docs.k3s.io/security/secrets-encryption
